mos-platform/common/c/math.cc

```diff
@@ -0,0 +1,22 @@
+#include <math.h>
+
+template <typename T> static inline bool _is_nan(const T x) { return x != x; }
+
+template <typename T> static inline T _fmax(const T x, const T y) {
+  if (_is_nan(x)) return y;
```

Comment:
This isn't needed; if `x` is nan, then `x > y` is false, so the ternary returns `y`. Replace this line with a comment to this effect.

Reply:
Nice catch - fixed.